doc/supportedPlatformVersions.md

```diff
@@ -1,336 +1,365 @@
 # Supported platforms and versions
```

Comment:
Any idea why this whole file is basically getting overwritten?

Reply:
It looks like the ./build/generateConstants.sh reshuffled those changes.
Before:
https://github.com/microsoft/Oryx/blob/22c6d63ac9f0953238e5a3a9d5bca50799a00a9b/doc/supportedPlatformVersions.md

After:
https://github.com/microsoft/Oryx/blob/77f446b1d2fa2b12c07a00988ea9d1aae4cc5757/doc/supportedPlatformVersions.md